examples/prefect/model_data_benchmark.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+import sys
+import time
+
+import numpy as np
+from sklearn import base
+
+import nnbench
+
+
+@nnbench.benchmark(name="Model size (bytes)", tags=("model-meta",))
+def modelsize(model: base.BaseEstimator) -> int:
+    model_bytes = pickle.dumps(model)
+    return sys.getsizeof(model_bytes)
```

Comment:
Is there any reason to not just call `sys.getsizeof()` on the model directly?

Reply:
Calling on the object directly does only give the memory footprint of the object itself, not of any other values it references. See [here](https://stackoverflow.com/questions/19516403/why-pickle-dumpobj-has-different-size-with-sys-getsizeofobj-how-to-save-var). Thats why I went with serializing first.